docker/alertmanager.yml

```
@@ -6,3 +6,11 @@ receivers:
 - name: "default"
   webhook_configs:
   - url: http://web:8000/api/v1/alerts
+    # Promgen requires authentication for this webhook endpoint.
+    # First, please access to Promgen's Admin site to create a service account specifically
+    # for Alertmanager. Next, grant this account the "alert_trigger" permission and generate
```

Promgen does not trigger alerts. Only processes them. I think we should call this permission `process_alert`.

```suggestion
    # for Alertmanager. Next, grant this account the "process_alert" permission and generate
```